test/fixtures/sinatra-app/features/error_reporting.feature

```diff
@@ -41,9 +41,11 @@ Scenario: Sinatra override context
 
 Scenario: Sinatra handles metadata
     # Tests the timestamp comparison
```

Comment:
Is this meant to be here?

Reply:
Yeah, I thought it was a good idea to mark which scenarios were tests.